.settings/org.eclipse.m2e.core.prefs

```diff
@@ -0,0 +1,4 @@
+activeProfiles=
```

Comment:
Committing editor preference files, maybe update .gitignore?

Reply:
Nope, that's intentional. This way, everyone gets consistent formatting, etc. Unfortunately the formatter isn't configured properly for this project, but that's a separate issue